src/Routes/RosSystemDetail/ros-details-page.scss

```diff
@@ -12,4 +12,7 @@
       }
     }
   }
+}
+.state-value-gap {
+  margin-right: 107px;
```

Comment:
@r14chandra can you please tell me how did you come up with this value for right margin?

Reply:
Just to make it consistent with the gap between `Provider`, `Instance type` and others key-value pairs, I tried to match it. It's achievable at `107px` for `State` key by setting it for margin-right property . Do you have any other suggestions?

Reply:
@r14chandra I have put the suggestion in the comments above, please check and let me know what you think!

Reply:
attaching the screenshot with the `DescriptionList` changes for your reference

![image](https://user-images.githubusercontent.com/5928530/155087748-de4e604b-c49c-4e15-87a3-1d35d616f2ef.png)